packages/admin-ui/src/pages/wifi/components/localProxying/LocalProxying.tsx

```diff
@@ -60,6 +64,25 @@ export function LocalProxying(): JSX.Element {
     }
   }
 
+  // Helper UI in case the HTTPs Portal is bad
+  if (dnpsRequest.data) {
+    const httpsPortalDnp = dnpsRequest.data.find(
+      dnp => dnp.dnpName === httpsPortalDnpName
+    );
+    if (
+      !httpsPortalDnp &&
+      dappnodeIdentity.internalIp !== dappnodeIdentity.ip
```

Comment:
This check needs way more explanation, you must add a big comment here. Is it even necessary tho to do this check? Consider:
- Receiving a boolean from the DAPPMANAGER backend for `isBehindRouter` or something
- Always should the banner